src/components/widgets/RecentListWidget.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+	<Card :loading="loading">
+		<template #title>
+			{{getNameByKey('recent')}} {{getNameByKey(title)}}
+		</template>
+
+		<template #control>
+			<router-link :to="listingPage">
+				<ButtonMore> {{getNameByKey('viewAll')}} </ButtonMore>
+			</router-link>
+		</template>
+
+		<template #body>
+			<b-container fluid>
+				<b-row>
+					<b-col
+						sm="6"
+						md="3"
+						lg="6"
+						xl="12"
+						v-for="(item, index) in recentList"
+						:key="`recent_${listingPage}_${index}_${item[header]}`"
+					>
+						<Card
+							class='card-item'
+							:item="item"
+						>
+							<template #header>
+								<router-link
+									:to="getItemHref(header, item[header])"
+									class="ex-title-text"
+									:title="getNameByKey(header) + ': ' + item[header]"
+								>
+									{{ item[header] }}
+								</router-link>
+							</template>
+							<template #body>
+								<div class="ex-row no-wrap"
+									v-for="(field, index) in fields"
+									:key="`field_${index}_${item[field]}`">
+									<div class="ex-text">
+										{{ getNameByKey(field) }}
+									</div>
+
+									<router-link
+										v-if="isKeyClickable(field)"
+										:to="getItemHref(field, item[field])"
+										class="ex-long-text"
+										:title="item[field]"
+									>
+										{{ item[field] }}
+									</router-link>
+
+									<div v-else class="ex-text">
+										{{ item[field] }}
+									</div>
+								</div>
+							</template>
+						</Card>
+					</b-col>
+				</b-row>
+			</b-container>
+		</template>
+	</Card>
+</template>
+
+<script>
+import Card from '@/components/containers/Card.vue';
+import ButtonMore from '@/components/controls/ButtonMore.vue';
+
+export default {
+	props: {
+		title: {
+			type: String,
+			require: true
+		},
+		listingPage: {
+			type: String,
+			require: true
+		},
+		dataGetter: {
+			type: String,
+			require: true
+		},
+		header: {
+			type: String,
+			require: true
+		},
+		fields: {
+			type: Array,
+			require: true
+		}
+	},
+
+	data() {
+		return {
+			clickableKeys: [
+				'ownerAddress'
+			]
+		};
+	},
+
+	components: {
+		Card,
+		ButtonMore
+	},
+
+	computed: {
```

Comment:
Should we handle the error state?

Reply:
I don't think it need because it's just an array from the list, default is an empty array